src/main/java/mate/academy/repository/cartitem/CartItemRepository.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.repository.cartitem;
+
+import mate.academy.model.CartItem;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
```

Comment:
not a mistake but remember that this annotation is optional when we extend JpaRepository